lib/api/src/entities/module/inner.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
     AsEngineRef,
 };
 
-/// IO errors that can happen while compiling a [`Module`].
+/// IO errors that can happen while compiling a [`BackendModule`].
```

Comment:
This should be Module, not BackendModule. `BackendModule` should be not be usable/public by the user

Reply:
I believe IoCompileError is an error for all backends... isn't it?

Reply:
Sure. This type shouldn't be there at all actually. Removing in next commit.